### DISCUSSION TITLE
Executing automations on click of button, by user

### DISCUSSION THREAD

**Discussion by demitchell14** [Budibase/budibase#5736] [Help]
It appears that only power users and above can trigger an automation via a button. I'm trying to make an automation that sends out an email whenever a button is clicked, and regular users need to be able to perform this action, but can't. The only solution I can think of to solve this, is creating an external email service which is annoying, considering the system works, my users just can't use it.

Is there some configuration that is changeable that I'm missing, or is this just how it is?

**Comment by mjashanks** [comment]
Hi @demitchell14 - I have reproduced this, and it's a bug. We'll look into it.

Thanks for reporting!
